subhub/tests/unit/stripe/utils.py

```diff
@@ -22,6 +22,11 @@ def send_message(QueueUrl={}, MessageBody={}):
         return {"ResponseMetadata": {"HTTPStatusCode": 200}}
 
 
+class MockSnsClient:
+    def publish(Message: dict = {}, MessageStructure: str = "json", TopicArn: str = ""):
```

Comment:
bug: don't assign keyword args mutable objects like dicts
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Interesting, OK I changed this to  have None for Message and TopicArn, tests still pass.